client/src/components/DataFiles/fixtures/DataFiles.systems.fixture.js

```diff
@@ -8,20 +8,23 @@ const systemsFixture = {
         api: 'tapis',
         icon: null,
         hidden: true,
+        homeDir: '/home/username',
       },
       {
         name: 'My Data (Frontera)',
         system: 'frontera.home.username',
```

Comment:
the `system` looks different than what we see (like it has the extra .home.username).  should we use this as an opportunity to just update the fixture completely?  There is also a `keyservice` and `default` property

Reply:
Yeah I added this field but the entire fixture would need to change to reflect the updated object. There is a task created for [Files Unit Tests](https://jira.tacc.utexas.edu/browse/TV3-101) that could cover this as well